entrypoint.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+DEBUG=${DEBUG:-"false"}
+SLEEP=${SLEEP:-"0"}
+
+TARGET_FILE=${TARGET_FILE:-""}
+TARGET_HOST=${TARGET_HOST:-"google.com"}
```

Comment:
small one - this seems a bit unsafe choice - instead of failing fast when neither file nor `TARGET_HOST` is specified, the script will run scraping `google.com`, which is unlikely to be what the user wanted.

Reply:
Valid argument, let me fix this.